# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/credentialmanagement/identity/PWebIdentity.ipdl
# Commit: d07def1365a4
# Full Hash: d07def1365a49dc8ecbf5b6c10fdfbd5ec750932
# Author: pstanciu <pstanciu@mozilla.com>
# Date: 2025-06-11 18:38:51
# Regressor Bug: 1969017
# File Overlap Count: 2
# Description:
#   Revert "Bug 1969015, 1969017: apply code formatting via Lando" for causing WPT failures @lfedcm-identity.create-store-collect.tentative.sub.https.html
#   
#   This reverts commit 03d4474e360f28be0fb075379f4e4535bd34b252.
#   
#   Revert "Bug 1969017 - Refactor FedCM to use a managed actor rather than tagging onto WindowGlobalParent - r=anti-tracking-reviewers,emz"
# ==============================================================================

diff -r 4e25ddaadc86 -r d07def1365a4 dom/credentialmanagement/identity/PWebIdentity.ipdl
--- a/dom/credentialmanagement/identity/PWebIdentity.ipdl	Wed Jun 11 13:42:13 2025 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,57 +0,0 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this file,
- * You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-/*
- * IPC Transaction protocol for the FedCM DOM API.
- */
-
-include protocol PWindowGlobal;
-
-include "mozilla/dom/IdentityCredentialSerializationHelpers.h";
-
-using mozilla::dom::CredentialMediationRequirement from "mozilla/dom/CredentialManagementBinding.h";
-using mozilla::dom::IdentityCredentialDisconnectOptions from "mozilla/dom/IdentityCredentialBinding.h";
-using mozilla::dom::IdentityCredentialRequestOptions from "mozilla/dom/IdentityCredentialBinding.h";
-using mozilla::dom::IdentityProviderConfig from "mozilla/dom/IdentityCredentialBinding.h";
-using mozilla::dom::LoginStatus from "mozilla/dom/LoginStatusBinding.h";
-
-
-namespace mozilla {
-namespace dom {
-
-struct IPCIdentityCredential
-{
-  nsString id;
-  nsString? token;
-};
-
-
-union WebIdentityGetCredentialResponse {
-  nsresult;
-  IPCIdentityCredential;
-};
-
-async protocol PWebIdentity {
-  manager PWindowGlobal;
-
-  parent:
-    async RequestCancel();
-
-  async GetIdentityCredential(IdentityCredentialRequestOptions aOptions,
-                              CredentialMediationRequirement aMediationRequirement,
-                              bool aHasUserActivation)
-    returns (WebIdentityGetCredentialResponse response);
-
-  async DisconnectIdentityCredential(IdentityCredentialDisconnectOptions aOptions) returns (nsresult rv);
-
-  async PreventSilentAccess() returns (nsresult rv);
-
-  async SetLoginStatus(LoginStatus foo) returns (nsresult rv);
-
-  child:
-    async __delete__();
-};
-
-}
-}